pkg/ui/workspaces/db-console/src/views/app/components/tenantDropdown/tenantDropdown.tsx

```diff
@@ -38,15 +38,15 @@ const TenantDropdown = () => {
     }
   };
 
-  if (tenants.length == 0) {
+  if (tenants.length < 2) {
     return null;
   }
 
   return (
     <ErrorBoundary>
       <Dropdown
         items={createDropdownItems()}
-        onChange={tenantID => onTenantChange(tenantID)}
+        onChange={(tenantID: string) => onTenantChange(tenantID)}
       >
         <div className="tenant-selected">{"Tenant " + currentTenant}</div>
```

Comment:
nit: @dhartunian had mentioned a few ideas for making this label more clear. one was to make it `Tenant: `